helm/kubernetes-namespace-prep/templates/authenticator-rolebinding.yml

```diff
@@ -0,0 +1,23 @@
+---
+{{- if .Values.authnRoleBinding.create -}}
+{{- $g := (lookup "v1" "ConfigMap" .Values.authnk8s.namespace .Values.authnk8s.goldenConfigmap).data -}}
+apiVersion: {{ include "conjur-prep.rbac-api" . }}
+kind: RoleBinding
+metadata:
+  name: {{ .Values.authnk8s.namespace }}-conjur-authenticator-role-binding
+  labels: 
+    app.kubernetes.io/name: {{ .Values.authnk8s.namespace }}-authn-k8s
```

Comment:
I believe that what I originally proposed in the Issue  for the Kubernetes standard labels is not quite correct usage of the labels. The documented example is here, for reference: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

So based on the documented example, I think it should be:
```
    app.kubernetes.io/name: "rolebinding"
    app.kubernetes.io/component: "authn-k8s-access-permissions"
    app.kubernetes.io/instance: {{ .Release.Namespace }}
    app.kubernetes.io/part-of: "authn-k8s-namespace-config"